include/deal.II/fe/fe_values.h

```diff
@@ -2026,7 +2026,7 @@ class FEValuesBase :
    *
    * @post <code>laplacians[q]</code> will contain the Laplacian of the field
    * described by fe_function at the $q$th quadrature point.
-   * <code>gradients[q][i][j]</code> represents the $(i,j)$th component of the
+   * <code>laplacians[q][i][j]</code> represents the $(i,j)$th component of the
    * matrix of second derivatives at quadrature point $q$.
```

Comment:
I agree, but you might still add that the Laplacian computes as the trace of the Hessian, i.e., the sum of the diagonal entries in the Hessian matrix.